packages/frontend/src/ee/features/aiCopilot/components/ChatElements/AiChartQuickOptions.tsx

```diff
@@ -59,6 +69,53 @@ export const AiChartQuickOptions = ({
         vizConfig,
     ]);
 
+    const onClickExplore = () => {
+        if (
+            user?.data?.userUuid &&
+            user?.data?.organizationUuid &&
+            projectUuid &&
+            agentUuid &&
+            metricQuery?.exploreName
+        ) {
+            track({
+                name: EventName.AI_AGENT_CHART_EXPLORED,
+                properties: {
+                    userId: user.data.userUuid,
+                    organizationId: user.data.organizationUuid,
+                    projectId: projectUuid,
+                    aiAgentId: agentUuid,
+                    threadId: message.threadUuid,
+                    messageId: message.uuid,
+                    tableName: metricQuery.exploreName,
+                },
+            });
+        }
+    };
+
+    const onConfirm = () => {
```

Comment:
heh I was thinking on doing this one on the server side because, at the moment, you can save the same viz every time 😅 
(once I finish #15609)